### PR TITLE
added 'areEqualArrays' and 'areFullEqualArrays' methods

### PR DESCRIPTION
**Pull request by inikonorov**

### CODE REVIEW

File: docs/documentation.js [hunk 1]
```diff
@@ -649,5 +649,17 @@ module.exports = {
       en: 'Checks if passed value is an object',
       pl: 'Sprawdza czy przekazana wartość jest obiektem.'
     }
+  },
+  areEqualArrays: {
```
Comment (pkonieczniak) [inline docs/documentation.js]: `areArraysEqual` I think that this method name is closer to English grammar.

File: docs/documentation.js [hunk 2]
```diff
@@ -649,5 +649,17 @@ module.exports = {
...
+    parameters: ['array1', 'array2'],
+    description: {
+      en: 'Checks if arrays are the same'
```
Comment (pkonieczniak) [inline docs/documentation.js]: Checks if the arrays have the same elements

File: docs/documentation.js [hunk 3]
```diff
@@ -649,5 +649,17 @@ module.exports = {
...
+    }
+  },
+  areFullEqualArrays: {
```
Comment (pkonieczniak) [inline docs/documentation.js]: Maybe we can name this method `areArraysStrictEqual`?

File: fixtures/input.js [hunk 4]
```diff
@@ -942,5 +942,23 @@ module.exports = {
         code: 'isPlainObject([])'
       }
     ]
+  },
+  areEqualArrays: {
+    examples:
+    [
+      {
+        input: [[], []],
```
Comment (pkonieczniak) [inline fixtures/input.js]: This file is responsible for documentation creation. 
You can see how it looks at https://buxlabs.pl/en/tools/js/pure-conditions/repl 

So I think that input should contains better example that user which read documentations can easily understand what the methods do.

File: test.js [hunk 5]
```diff
@@ -557,3 +559,13 @@ deepStrictEqual(isPlainObject({ foo: 'bar' }), true)
 deepStrictEqual(isPlainObject(function () {}), false)
 deepStrictEqual(isPlainObject([]), false)
 deepStrictEqual(isPlainObject(null), false)
+
+deepStrictEqual(areEqualArrays([], []), true)
+deepStrictEqual(areEqualArrays([1, 2, 3], [1, 2, 3]), true)
```
Comment (pkonieczniak) [inline test.js]: Could you add more test cases? Mixing letters and numbers, etc.

File: index.js [hunk 6]
```diff
@@ -411,5 +411,17 @@ module.exports = {
   },
   isPlainObject (object) {
     return Object.prototype.toString.call(object) === '[object Object]'
+  },
+  areEqualArrays(array1, array2) {
+    if (array1.length !== array2.length) {
```
Comment (pkonieczniak) [inline index.js]: It should not work for more complex cases but for now is enough.

File: index.js [hunk 7]
```diff
@@ -411,5 +411,17 @@ module.exports = {
...
+      return false;
+    }
+
+    const newArray1 = array1.slice().sort();
+    const newArray2 = array2.slice().sort();
+    return newArray1.every((value, index) => value === newArray2[index]);
```
Comment (pkonieczniak) [inline index.js]: It should not work for more complex cases but for now is enough.

**Comment by pkonieczniak** [comment]
Looks good. Some small improvements and we can merge your pr.
At the end please run `npm run lint:fix` and `npm test` command. This information is not in readme, so I will add it.

**Comment by inikonorov** [comment]
can you give me translations for these methods in polish?

**Comment by emilos** [comment]
@inikonorov please feel free to put "TODO" as a translation, we'll make it optional in other PR

**Comment by inikonorov** [comment]
done!

**Comment by emilos** [comment]
@inikonorov thanks for the help! I've released the recent changes as `v1.0.2`

https://buxlabs.pl/en/tools/js/pure-conditions/repl#areArraysEqual

**Pull request merged**